module/Company/src/Company/Service/Company.php

```diff
@@ -183,87 +207,193 @@ public function getCategoryList($visible)
                 $translator->translate('You are not allowed to list all categories')
             );
         }
-        if ($visible) {
-            $categories = $this->getCategoryMapper()->findVisibleCategoryByLanguage($translator->getLocale());
-            $jobsWithoutCategory = $this->getJobMapper()->findJobsWithoutCategory($translator->getLocale());
-            $filteredCategories =  $this->filterCategories($categories);
-            $noVacancyCategory = count(array_filter($filteredCategories, function ($el) {
-                return $el->getSlug() == "jobs";
-            })) ;
-            if (count($jobsWithoutCategory) > 0 && $noVacancyCategory  == 0) {
-                $filteredCategories[] = $this->getCategoryMapper()
-                    ->createNullCategory($translator->getLocale(), $translator);
+        $categories = $this->getCategoryMapper()->findVisibleCategoryByLanguage($translator->getLocale());
+        $jobsWithoutCategory = $this->getJobMapper()->findJobsWithoutCategory($translator->getLocale());
+        $filteredCategories =  $this->filterCategories($categories);
+        $noVacancyCategory = count(array_filter($filteredCategories, function ($el) {
+            return $el->getSlug() == "jobs";
+        })) ;
+        if (count($jobsWithoutCategory) > 0 && $noVacancyCategory  == 0) {
+            $filteredCategories[] = $this->getCategoryMapper()
+                ->createNullCategory($translator->getLocale(), $translator);
+        }
+        return $filteredCategories;
+    }
+
+    public function getLabelList($visible)
```

Comment:
The parameter name is a little confusing. I would expect `$visible = false` to return all labels that aren't visible. Instead, it returns all labels (if the user has admin rights). Consider renaming the parameter to something more descriptive such as `$onlyVisible`.

Reply:
I actually think that visible is more descriptive than onlyVisible. I added some PHPDoc

Reply:
Adding documentation is already a good improvement 👍